src/vec/drain.rs

```diff
@@ -19,25 +21,25 @@ use super::VecView;
 /// use heapless::{vec, Vec};
 ///
 /// let mut v = Vec::<_, 4>::from_array([0, 1, 2]);
-/// let iter: vec::Drain<'_, _> = v.drain(..);
+/// let iter: vec::Drain<'_, _, _> = v.drain(..);
```

Comment:
This example is kind of useless, might as well remove it given `Vec::drain` has a better one.

Reply:
True, removed.